hack/allocate.sh

```diff
@@ -242,6 +243,26 @@ EOF
 
 }
 
+tekton() {
+  echo "${em}⑧ Tekton Pipelines and build tasks${me}"
+
+  # Install latest tekton pipelines release
+  kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
```

Comment:
The GitHub CI environment is.... resource challenged.  This might need generous default waits and explicit long-timeout `kubectl wait ....` commands before these will complete.